deploy/clusterrole.yaml

```diff
@@ -0,0 +1,29 @@
+kind: ClusterRole
+metadata:
+  creationTimestamp: null
+  name: acme-controller
+  annotations:
+    openshift.io/reconcile-protect: "true"
+rules:
+- apiGroups:
+  - ""
```

Comment:
I saw it the docs with null (https://docs.openshift.org/latest/admin_guide/manage_authorization_policy.html#manage-authorization-policy-creating-local-role) but I don't have a strong opinion on it